src/main/java/constants/SkillingNodeType.java

```diff
@@ -0,0 +1,25 @@
+package constants;
+
+//This enum contains a list of the different types
+//of skilling nodes
+public enum SkillingNodeType {
+	//Maybe do something special for motherlode mine?
+	CLAY, ESSENCE, COPPER_ORE, TIN_ORE, LIMESTONE, BLURITE_ORE, IRON_ORE,
+	DAEYALT_ORE, SILVER_ORE, VOLCANIC_ASH, COAL_ORE, PAYDIRT, SANDSTONE,
+	DENSE_ESSENCE, GEM_ROCK, GOLD_ORE, VOLCANIC_SULPHUR, GRANITE, MITHRIL_ORE,
+	LOVAKITE_ORE, ADAMANTITE_ORE, SALT_MINE, RUNITE_ORE, AMETHYST_ORE,
+	ANVIL,
+	//Implement special fishing spots like karambwan fishing?
+	NET_BAIT, LURE_BAIT, BIG_NET, HARPOON_CAGE,
+	TREE, ACHEY_TREE, OAK_TREE, WILLOW_TREE, TEAK_TREE, MAPLE_TREE, HOLLOW_TREE,
+	MAHOGANY_TREE, ARCTIC_PINE_TREE, YEW_TREE, SULLIUSCEP, MAGIC_TREE, REDWOOD_TREE,
+	AGILITY_COURSE, AGILITY_SHORTCUT,
+	//How to implement thieving?
+	THIEVABLE_NPC, THIEVABLE_STALL, THIEVABLE_CHEST, THIEVABLE_DOOR,
+	//Slayer?
+	//Special farming patches?
+	ALLOTMENT, FLOWER_PATCH, HERB_PATCH, HOPS_PATCH, BUSH_PATCH, TREE_PATCH, FRUIT_TREE_PATCH,
+	//Construction?
+	BIRD_SNARE, TRACKING, DEADFALL, FALCONRY, BOX_TRAPPING, BUTTERFLY_NETTING, PITFALL_TRAPPING,
+	NET_TRAPPING, IMPLING, RABBIT_SNARING, MAGIC_BOX_TRAPPING
```

Comment:
Need to change these into the hunter specific thing that would be located within the chunk, not the tool used.
I.e. Red-chinchompa, swamp lizard, birdhouse space etc.